thunder.profile

```diff
@@ -15,7 +15,7 @@ use Drupal\block\Entity\Block;
 function thunder_system_info_alter(array &$info, Extension $file, $type) {
   // Thunder can not work properly without these modules. So they are enforced
   // to be enabled.
-  $required_modules = ['config_selector'];
+  $required_modules = ['config_selector', 'views', 'media_entity', 'node'];
```

Comment:
Like it. Profiles having dependencies ftw!

Reply:
Not sure about media_entity though, the dependency will change, once we switch to core media. But that should not be a problem.